app/infrastructure/database/orm/query_job_orm.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# http://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/Timestamps.html
```

Comment:
Please remove this commented-out code.

Reply:
Issue solved. Please refer to this line of the [commit](https://github.com/KeroKnows/s-killer/commit/c1d43b9257897b20edca40169e25f234a0879b85#diff-035149020368d3a3eefdf2929c3f5b5054917c0d2bc787a0e7061a2f3bdb6014L3).